weave-js/src/components/RadixTooltip/index.tsx

```diff
@@ -57,12 +57,13 @@ export const Portal = RadixTooltip.Portal;
  */
 export const Content = React.forwardRef(
   ({className, children, ...props}: RadixTooltip.TooltipContentProps, ref) => (
-    <Tailwind>
+    <Tailwind style={{display: 'contents'}}>
       <RadixTooltip.Content
         className={twMerge(
           'night-aware',
-          'rounded text-sm leading-[140%] shadow-md',
-          'bg-moon-900 p-12 text-moon-200 dark:bg-moon-50 dark:text-moon-800'
+          'z-[999999] rounded text-sm leading-[140%] shadow-md',
```

Comment:
had to add this ridiculous z-index value otherwise tooltips will be rendered below the app's top nav bar. this wasn't a problem when we were using the semantic tooltip because they set this _even more_ ridiculous z-index

<img width="481" alt="Screenshot 2024-07-25 at 6 44 18 PM" src="https://github.com/user-attachments/assets/3704b191-bd63-47b7-9631-1bfa138b3fe4">